jenkins/opensearch-dashboards/integ-test-opensearch-dashboards.jenkinsfile

```diff
@@ -0,0 +1,83 @@
+lib = library(identifier: 'jenkins@20211123', retriever: legacySCM(scm))
```

Comment:
What is this identifier, why use fixed date value?

Reply:
Jenkins caches the library sometimes, so when we make code changes we tend to increment this everywhere.